File: offers/back/Dbo/Candidates/Experience.cs
using System;
using System.Collections.Generic;

namespace AiOffer.Dbo.Candidates
{
    public partial class Experience
    {
        public ulong Id { get; set; }
        public ulong IdCandidate { get; set; }
        public string Title { get; set; }
        public string Description { get; set; }
        public DateTime Start { get; set; }
        public DateTime? End { get; set; }

        public virtual Candidate IdCandidateNavigation { get; set; }
    }
}

File: offers/front/src/app/models/experience.ts
import { ApiExperience } from './api/api-experience';

export class Experience {
    public readonly id: bigint;
    public readonly title: string;
    public readonly description: string;
    public readonly start: string;
    public readonly end: string;

    constructor(json: ApiExperience) {
        this.id = json.id;
        this.title = json.title;
        this.description = json.description;
        this.start = json.start;
        this.end = json.end;
    }
}